packages/histoire/src/node/vite.ts

```diff
@@ -78,10 +78,11 @@ export async function mergeHistoireViteConfig (viteConfig: InlineConfig, ctx: Co
   let flatPlugins = []
   if (viteConfig.plugins) {
     for (const pluginOption of viteConfig.plugins) {
-      if (Array.isArray(pluginOption)) {
-        flatPlugins.push(...await Promise.all(pluginOption))
+      const resolvedPluginOption = await pluginOption
```

Comment:
I don't understand why this changed. Now if it's an array it's being awaited twice

Reply:
Ok. I guess awaiting twice is actually correct: https://github.com/vitejs/vite/blob/af86e5bcfe4b78f486f499cba09c3270fb151d54/packages/vite/src/node/config.ts#L111

Reply:
A vite plugin can be of the form: `[promise, promise, promise]`